test/remote/gateways/remote_adyen_test.rb

```diff
@@ -607,6 +607,14 @@ def test_successful_store
     assert_equal 'Authorised', response.message
   end
 
+  def test_successful_tokenize_only_store
```

Comment:
It's great to have a validating remote test!

In keeping with Active Merchant guidelines, could we also add an equivalent unit test. For folks who may not have sandbox credentials, the canned response recorded in unit tests could serve as a useful reference at some point.